bottom-bar/src/main/java/com/roughike/bottombar/BottomNavigationBehavior.java

```diff
@@ -65,6 +65,13 @@ public boolean onDependentViewChanged(CoordinatorLayout parent, V child, View de
 
     @Override
     public void onDirectionNestedPreScroll(CoordinatorLayout coordinatorLayout, V child, View target, int dx, int dy, int[] consumed, @ScrollDirection int scrollDirection) {
+        if (scrollDirection == ScrollDirection.SCROLL_NONE) {
```

Comment:
and are you sure this is a safe thing to do? because if the system hasn't set it to scrolling there must be a reason, right? what happens if someone consumes the scroll before you, or if there is a touchintercept, what happen on those cases?